src/main/java/com/puppycrawl/tools/checkstyle/DetailNodeTreeStringPrinter.java

```diff
@@ -182,13 +182,17 @@ private static DetailAST createFakeBlockComment(String content) {
         blockCommentBegin.setType(TokenTypes.BLOCK_COMMENT_BEGIN);
         blockCommentBegin.setText("/*");
         blockCommentBegin.setLineNo(0);
-        blockCommentBegin.setColumnNo(0);
+        // -3 indentation because of BLOCK_COMMENT_BEGIN and leading astrisk
+        final int blockCommentIndent = -3;
+        blockCommentBegin.setColumnNo(blockCommentIndent);
 
         final DetailAST commentContent = new DetailAST();
         commentContent.setType(TokenTypes.COMMENT_CONTENT);
         commentContent.setText("*" + content);
         commentContent.setLineNo(0);
-        commentContent.setColumnNo(2);
+        // javadoc should starts at 0 column, so comment content
+        // so comment content with leading asterisk has -1 column
```

Comment:
@romani 
````
 |--BLOCK_COMMENT_BEGIN -> /* [1:0]
 |   |--COMMENT_CONTENT -> *\n* class javadoc\n [1:2]
````
First `*` in `COMMENT_CONTENT` is leading asterisk. AST splits `*` from `/*` in javadoc start (`/**`).